<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>TabWidget</class>
 <widget class="QTabWidget" name="TabWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>591</width>
    <height>324</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>TabWidget</string>
  </property>
  <property name="currentIndex">
   <number>2</number>
  </property>
  <widget class="QWidget" name="Filtering">
   <attribute name="title">
    <string>Filtering</string>
   </attribute>
   <layout class="QGridLayout" name="gridLayout_3">
    <item row="0" column="0">
     <layout class="QGridLayout" name="filter_layout"/>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="tab">
   <attribute name="title">
    <string>Equalization</string>
   </attribute>
   <layout class="QVBoxLayout" name="verticalLayout_2">
    <item>
     <layout class="QVBoxLayout" name="equalizer_layout"/>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="Bourder_trecing">
   <attribute name="title">
    <string>Bourder_trecing</string>
   </attribute>
   <layout class="QVBoxLayout" name="verticalLayout_3">
    <item>
     <layout class="QVBoxLayout" name="Boarder_tracing_layout"/>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
